src/core/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+//    "composite": true,
+    "outDir": "./target",
```

Comment:
not used at the moment, going to use it in the next PR. Should I remove it?

Reply:
I guess comment it out, just in case that PR gets delayed for some reason.